res/values/strings.xml

```diff
@@ -13748,4 +13756,19 @@
 
     <string name="battery_share_description">Charge other devices by placing them on the back of your phone</string>
     <string name="battery_share">Battery share</string>
+
+    <string name="choose_duress_password_title">Choose a new Duress Lock</string>
+    <string name="update_duress_password_title">Update existing Duress Lock</string>
+
+    <string name="duress_pin_exist">PIN</string>
+    <string name="duress_password_exist">PASSWORD</string>
+    <string name="duress_pin_and_password_exist">PIN &amp; PASSWORD</string>
+    <string name="added_a_duress_pin_and_password">NONE</string>
+
+    <string name="duress_descriptions">Duress mode in GrapheneOS will only work on if you are using a lockscreen which is the same as what you have configured for duress. Duress mode will wipe the device fully, including user data and eSIMS. Duress mode does not erase the presence of GrapheneOS of the device. After duress mode has been triggered by the user, the device will act as if it is a fresh installation of GrapheneOS.</string>
```

Comment:
Fix: Remove `on` from `will only work on if`
Fix: `eSIMS` should be `eSIMs`
Fix: `GrapheneOS of the device` should be `GrapheneOS on the device`
Change `lockscreen which is the same` to `screen lock that matches`
Change `device will act` to `device will function`

The wording for some things can also be improved but I am not sure what info is / isn't important to show here.